README.md

```diff
@@ -39,4 +39,27 @@ Pair programming is therefore a good alternative to development alone. It allows
 
 [Read More](https://en.wikipedia.org/wiki/Pair_programming)
 
----
\ No newline at end of file
+---
+### Pull request
+
+**What's a pull request ?**
+
+A pull request in programming could be defined as a way to ask for a code review on a feature or a piece of code we have done and on which we want a feedback from our colleagues. It's a very useful and a common way in programming to ensure that the code is working correctly, following the coding styles and convention that have been chose for the project and mostly that the piece of code is able to be added to the existing one without any problem.
+
+**How to create a pull request in GitHub**
+
+First, you need to commit and push a piece of code on the branch you are working on. This can be done either on GitHub directly, by using and IDE or even using terminal with Git command.
```

Comment:
I think "and IDE" could be replaced by "an"

Reply:
Yes i'll fix it